specs/order_spec.rb

```diff
@@ -78,33 +80,43 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+
+describe "Order Wave 2" do
   describe "Order.all" do
-    it "Returns an array of all orders" do
-      # TODO: Your test code here!
-    end
+    it "Returns an array of all orders" do # DONE
+      Grocery::Order.all.must_be_kind_of Array
+    end # it/do
 
-    it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
-    end
 
-    it "Returns accurate information about the last order" do
-      # TODO: Your test code here!
-    end
-  end
+    it "Returns accurate information about the first order" do # DONE
+      Grocery::Order.all[0].id.must_equal 1
+      product_in_1st = {"Slivered Almonds" => 22.88, "Wholewheat flour" => 1.93, "Grape Seed Oil" => 74.9}
+      Grocery::Order.all[0].products.must_equal product_in_1st
+    end # it/do
+
+    it "Returns accurate information about the last order" do #DONE
+      Grocery::Order.all[-1].id.must_equal 100
+      product_last = {"Allspice" => 64.74, "Bran" => 14.72, "UnbleachedFlour" => 80.59}
+      Grocery::Order.all[-1].products.must_equal product_last
+    end # it/do
+  end # describe "Order.all" do
+
 
   describe "Order.find" do
-    it "Can find the first order from the CSV" do
-      # TODO: Your test code here!
+    it "Can find the first order from the CSV" do # DONE
+      first_id = Grocery::Order.find(1)
+      first_id.id.must_equal 1
```

Comment:
I'm not sure about the name of this variable `first_id`. Rather than an id, I think it holds an instance of `Order`, so `first_order` would be more appropriate. That makes the last line make more sense too: `first_order.id.must_equal 1`.